pagerduty/table_pagerduty_incident.go

```diff
@@ -175,6 +180,26 @@ func tablePagerDutyIncident(_ context.Context) *plugin.Table {
 				Description: "The teams involved in the incident's lifecycle.",
 				Type:        proto.ColumnType_JSON,
 			},
+			{
+				Name:        "service_id",
```

Comment:
Could you please arrange the columns by column type, moving the JSON columns towards the bottom and placing the other columns towards the top?